Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -105,7 +105,7 @@ inline void ComputeStrokeStyle(ID2D1DeviceContext* deviceContext) {
         deviceContext->GetFactory(&factory);
 
         std::vector<float> adjustedDashes(dashes.size());
-        std::transform(dashes.cbegin(), dashes.cend(), adjustedDashes.begin(), [this](const float& f) -> float { return f / lineWidth; });
+        std::transform(dashes.cbegin(), dashes.cend(), adjustedDashes.begin(), [this](const CGFloat& f) -> float { return f / lineWidth; });
```

Comment:
It complained when I attempted to capture a member by reference. Don't think I didn't try ;P

---
In reply to: [85365904](https://github.com/Microsoft/WinObjC/pull/1240#discussion_r85365904) [](ancestors = 85365904)